js/actions/windowActions.js

```diff
@@ -216,13 +231,15 @@ const windowActions = {
   /**
    * Dispatches a message to the store when the frame is active and the window is focused
    *
-   * @param {Object} frameProps - the frame properties for the webview in question.
+   * @param {Object} location - location for the webview in question.
+   * @param {Object} tabId - tabId for the webview in question.
    */
-  setFocusedFrame: function (frameProps) {
-    if (frameProps) {
+  setFocusedFrame: function (location, tabId) {
+    if (location) {
```

Comment:
I think this logic should be in the reducer. `setFocusedFrame` should only need `tabId`